docs/installation.rst

```diff
@@ -5,14 +5,16 @@
 Installation
 ============
 
-To install |project_name|, you need `Python 3`_ and `pip`_ installed. You
-can then either
-`download it directly
+To install Borg-Import, you need `Python 3` and `pip` installed. You
```

Comment:
Yeah, guess these tools are very popular / well known and do not need that linking.

But then also check if there are other links via a global search and also remove the entry in `global.rst.inc`.

Reply:
I'm happy to relink them. I removed them because the links were broken.